uaclient/livepatch.py

```diff
@@ -130,9 +130,9 @@ def status() -> Optional[LivepatchStatusStatus]:
         out, _ = system.subp(
             [LIVEPATCH_CMD, "status", "--verbose", "--format", "json"]
         )
-    except exceptions.ProcessExecutionError:
+    except exceptions.ProcessExecutionError as e:
         LOG.warning("canonical-livepatch returned error when checking status")
-        return None
+        raise e
```

Comment:
While you're improving this, can you add the error message to this logging statement too, so we get a record in bug reports